### PR TITLE
Update contribution

### PR DESCRIPTION
**Pull request by Soitora**
Freshen this page up a little.

Might be some wordings that could be changed.

### CODE REVIEW

File: src/help/contribution.md [hunk 1]
```diff
@@ -1,24 +1,50 @@
...
+### Website
+The repository that hosts this very website you're reading on now.
+- [tachiyomiorg/website](https://github.com/tachiyomiorg/website) ![GitHub issues](https://img.shields.io/github/issues/tachiyomiorg/website?style=social)
 
 ## Translation
+<a href="https://hosted.weblate.org/engage/tachiyomi/?utm_source=widget"><img src="https://hosted.weblate.org/widgets/tachiyomi/-/svg-badge.svg" alt="Translation status"/></a>
 
-**The project is available at [https://hosted.weblate.org/engage/tachiyomi/](https://hosted.weblate.org/engage/tachiyomi/)**
-(direct [https://hosted.weblate.org/projects/tachiyomi/strings/](https://hosted.weblate.org/projects/tachiyomi/strings/))
+Want to help translate the app to your language? You can do easily help do so using a service we use called **Weblate**.
```
Comment (arkon) [inline src/help/contribution.md:30]: "You can do easily help do so" ?

File: src/help/contribution.md [hunk 2]
```diff
@@ -1,24 +1,50 @@
...
+* [Subscriptions](https://docs.weblate.org/en/latest/user/profile.html#subscriptions)
+* [Glossary](https://docs.weblate.org/en/latest/user/translating.html#glossary)
+
+#### Project status
+***
+<a href="https://hosted.weblate.org/engage/tachiyomi/?utm_source=widget">
+<img src="https://hosted.weblate.org/widgets/tachiyomi/-/horizontal-auto.svg" alt="Translation status" />
+</a>
+
+## Monetary
```
Comment (arkon) [inline src/help/contribution.md:47]: "Donation"?

File: src/help/contribution.md [hunk 3]
```diff
@@ -1,24 +1,50 @@
...
+<a href='https://ko-fi.com/M4M8PGJI' target='_blank'><img height='36' style='border:0px;height:36px;' src='https://cdn.ko-fi.com/cdn/kofi1.png?v=2' border='0' alt='Buy Me a Coffee at ko-fi.com' /></a>
```
Comment (arkon) [inline src/help/contribution.md:48]: https://web.dev/external-anchors-use-rel-noopener/

File: src/help/contribution.md [hunk 4]
```diff
@@ -1,24 +1,50 @@
...
 
-[![Translation status](https://hosted.weblate.org/widgets/tachiyomi/-/multi-auto.svg)](https://hosted.weblate.org/engage/tachiyomi/?utm_source=widget)
+If you can't contribute code or translation but you still wish to help, then you can choose to contribute directly to the projects founder, [Inorichi](https://github.com/inorichi/).
```
Comment (arkon) [inline src/help/contribution.md:50]: "translations"

File: src/help/contribution.md (hunk 1 above)
Comment (Soitora) [reply src/help/contribution.md:30]: Probably a result from me adding/removing segments from around that string and me missing it didn't make any sense ^^

**Pull request merged**